### DISCUSSION TITLE
tenant名称是大小写不敏感的吗？

### DISCUSSION THREAD

**Discussion by huxiyu** [apache/pulsar#22865] [Q&A]
我新建个tenant名称是大写的，bin/pulsar-admin tenants list 是大写的，没问题。
但在trino里 show schemas in pulsar; 是小写的。

租户和命名空间是大小写不敏感的吗？

**Comment by huxiyu** [comment]
大写有问题，我强制使用小写了。

**Comment by hezhangjian** [comment]
I think it's sensitive
